pkgs/applications/misc/fff/default.nix

```diff
@@ -1,24 +1,30 @@
-{ stdenv, fetchFromGitHub, makeWrapper, xdg_utils, file, coreutils }:
+
+{ stdenv, fetchFromGitHub, makeWrapper,
+  bashInteractive, xdg_utils, file, coreutils, w3m, xdotool,
+  imageDisplaySupport ? true
```

Comment:
I'm not sure if w3m/xdotool depends on X, and if it does, then installing fff on a headless machine would pull in X, which doesn't really make sense for a text based file manager.

Reply:
Yet it includes `xdg_utils` unconditionally. My idea was that if you don't use and test this option you probably don't need to add it for the sake of "just in case". Figuring out that some of these dependencies are optional is not so difficult for the end user, I believe. And then it can be always overridden like `fff.override { xdg_utils = null; w3m = null; xdotool = null; }`.